app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(52, 6)) == 52, "Sum of parts should be equal to value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (split_integer(8, 2) == [4, 4]), f"Array should split value 8 on two equal parts not {split_integer(8, 2)}"
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert (split_integer(9, 1) == [9]), f"When number of parts equal to 1, one part should be equal to value 9"
```

Comment:
Similar to the previous comment, consider storing the result of `split_integer(9, 1)` in a variable if you plan to use it multiple times in the assertion or message.